packages/__docs__/src/Icons/theme.ts

```diff
@@ -22,12 +22,14 @@
  * SOFTWARE.
  */
 
+import type { Theme } from '@instructure/ui-themes'
+
 /**
  * Generates the theme object for the component from the theme and provided additional information
  * @param  {Object} theme The actual theme object.
  * @return {Object} The final theme object with the overrides and component variables
  */
-const generateComponentTheme = (theme) => {
+const generateComponentTheme = (theme: Theme) => {
```

Comment:
You can tell TypeScript that you expect `IconsTheme` as a return type.

```suggestion
const generateComponentTheme = (theme: Theme): IconsTheme => {
```